riscv/src/register/mcycle.rs

```diff
@@ -1,4 +1,6 @@
 //! mcycle register
 
 read_csr_as_usize!(0xB00);
+write_csr_as_usize!(0xB00);
 read_composite_csr!(super::mcycleh::read(), read());
+write_composite_csr!(|bits| super::mcycleh::write(bits), |bits| write(bits));
```

Comment:
Do you have any reason for not doing something like this?

```suggestion
write_composite_csr!(super::mcycleh::write, write);
```